www/src/data/sidebars/doc-links.yaml

```diff
@@ -309,6 +309,8 @@
                   link: /docs/centralizing-your-sites-navigation/
                 - title: Rendering Sidebar Navigation Dynamically*
                   link: /docs/rendering-sidebar-navigation-dynamically/
+            - title: Using Client-Side Only Packages
+            - link: /docs/using-client-side-only-libraries
```

Comment:
Should we also rename the file to `/docs/using-client-side-only-packages` ?

Reply:
Good point - I think that makes sense.  I'll get that pushed up.